File: phase_gate_two/task_one.py
def check_for_negative_number(user_input):
    if user_input > 3000 or user_input < 1000 or isinstance(user_input, str):
        return "Invalid input must be a number between 1000 - 3000"
    result = []
    for i in str(user_input):
        if int(i) % 2 == 0:
            result.append(True)
        else:
            result.append(False)
    return False if False in result else True


user_input = int(input("Enter a digit from 1000 to 3000: "))
print(check_for_negative_number(user_input))

File: phase_gate_two/java/TaskOne.java
public class TaskOne {
    
    public static void main(String[] args) {
        checkForNegativeNumber(2244);
    }

    public static void checkForNegativeNumber(int userInput) {
        if(userInput < 1000 || userInput > 3000) {
            System.out.println("Invalid input must be a number between 1000 - 3000");
        }
        String strNumber = Integer.toString(userInput);
        char[] charArray = strNumber.toCharArray();
        boolean result = true;
        for (int i = 0; i < charArray.length; i++) {
            if(charArray[i] % 2 != 0) {
                result = false;
                break;
            }
        }
        System.out.print(result == true ? "True" : "False");
    }
}
